packages/account-kit/cli/utils/deployGasTank.ts

```diff
@@ -0,0 +1,54 @@
+import { existsSync } from "fs";
+import { $, execa } from "execa";
+import { join, dirname } from "path";
+import { deployerClient, devPrivateKey } from "./deployLocal";
+import worlds from "@latticexyz/gas-tank/worlds.json" assert { type: "json" };
+
+const accountKitPath = join(dirname(process.argv[1]), "..");
```

Comment:
what does this path represent? how is the arg used?

Reply:
`process.argv[1]` is the path to the currently running script (`process.argv[2]` would be the first user provided arg). Added a comment to make this clearer.